pkg/vm/engine/tae/catalog/command.go

```diff
@@ -351,6 +297,9 @@ func (cmd *EntryCommand) WriteTo(w io.Writer) (n int64, err error) {
 			return
 		}
 		sn, err = cmd.Block.WriteTo(w)
+		if err != nil {
```

Comment:
It this, all about Serialize and DeSerialize?    Should you consider get rid of all these code and use protocol buffer?